terraform.tf.j2

```diff
@@ -103,3 +113,17 @@ data "terraform_remote_state" "security_tools" {
     dynamodb_table = "remote_state_locks"
   }
 }
+
+data "terraform_remote_state" "management" {
+  backend   = "s3"
+  workspace = lookup(local.slave_peerings, local.environment) == "management-dev" ? "default" : "management"
+
+  config = {
+    bucket         = "{{terraform.state_file_bucket}}"
+    key            = "terraform/dataworks/management.tfstate"
+    region         = "{{terraform.state_file_region}}"
+    encrypt        = true
+    kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
+    dynamodb_table = "remote_state_locks"
+  }
+}
```

Comment:
have you checked the terraform hierarchy on the `aws-common-infra` wiki to ensure that this repo can consume this new remote state?